war/src/main/js/widgets/config/tabbar.less

```diff
@@ -299,42 +299,42 @@
         position:relative;
       }  
       .repeatable-delete {
-        position:absolute;
-        top:-2px;
-        right:35px;
-        height:18px;
-        width:32px;
-        .border-radius-bottom(3px);
```

Comment:
No, it inherited a border-radius of 3px, it looks like. I think this was overriding it with a _bottom_ only a 3px border radius.